legacy/src/app/partials/nodedetails/deploy-options.html

```diff
@@ -4,78 +4,65 @@
     <p>Customise options</p>
   </div>
   <div class="col-9">
-    <ul class="p-list p-inline-list">
-      <li class="p-inline-list__item">
-        <div
-          class="p-form-group p-form-validation u-sv-1 u-display-inline-block"
+    <div
+      class="p-form-group p-form-validation"
+    >
+      <div class="p-form__control u-clearfix">
+        <input
+          id="installKVM"
+          type="checkbox"
+          data-ng-model="deployOptions.installKVM"
+          ng-disabled="!nodesManager.canBeKvmHost(osSelection)"
+        />
+        <label
+          for="installKVM"
+          class="u-float--left"
+          style="width: auto; margin-right: 1rem;"
         >
-          <div class="p-form__control u-clearfix">
-            <input
-              id="installKVM"
-              type="checkbox"
-              data-ng-model="deployOptions.installKVM"
-              ng-disabled="!nodesManager.canBeKvmHost(osSelection)"
-            />
-            <label
-              for="installKVM"
-              class="u-float--left"
-              style="width: auto; margin-right: 1rem;"
-            >
-              Register as MAAS KVM host (Ubuntu 18.04 LTS required)
-            </label>
+          Register as MAAS KVM host (Ubuntu 18.04 LTS required).&nbsp;
+          <a
+            class="p-link--external"
+            href="https://maas.io/docs/kvm-introduction"
+            target="_blank"
+          >
+            Read more
+          </a>
+        </label>
 
-            <p
-              class="u-sv-1 u-no-max-width"
-              ng-if="!nodesManager.canBeKvmHost(osSelection)"
-            >
-              <i class="p-icon--warning"></i>&nbsp;Ubuntu 18.04 is required to
-              create a KVM host.
-            </p>
-          </div>
-        </div>
-      </li>
-      <li class="p-inline-list__item">
-        <a
-          class="p-link--external"
-          href="https://maas.io/docs/kvm-introduction"
-          target="_blank"
+        <p
```

Comment:
Do you think we can remove this `p` block? It's not in the react version and it seems redundant to show a warning when the checkbox warns the same thing.

Reply:
I think you're right, pretty sure it was removed from the react version at some point, but not removed here.